ui/app/templates/vault/cluster/settings/mount-secret-backend.hbs

```diff
@@ -52,11 +52,29 @@
       <div class="field">
         <div class="b-checkbox">
           {{input type="checkbox" id="versioned" name="versioned" checked=versioned}}
-          <label for="versioned" class="is-label">
-            Versioned
+          <label for="version" class="is-label">
+            Version
           </label>
+
+          <div class="control is-expanded">
+            <div class="select is-fullwidth">
+              <select
+                 id="backend-type"
+                 value={{selectedType}}
+                 onchange={{action "onTypeChange" value="target.value"}}
+                data-test-secret-backend-type=true
+              >
+                {{#each (array 0 2) as |versionOption|}}
+                  <option selected={{eq version versionOption}} value={{versionOption}}>
+                  Version {{versionOption}}
+                  </option>
+                {{/each}}
+              </select>
+            </div>
+          </div>
+
           <p class="help has-text-grey">
-            The KV Secrets engine can operate in versioned or non-versioned mode. <em>Non-versioned may be upgraded to versioned later, but the opposite is not true</em>.
+            The KV Secrets engine can operate in different modes. 
```

Comment:
Can we describe what features are different? Not sure how helpful this message is otherwise

Reply:
Haha, yeah that was a bit hasty, sorry.